public/stylesheets/style.css

```diff
@@ -10,4 +10,24 @@ header {
 a {
   color: #00B7FF; }
 
+* {
+  margin: 0;
+  padding: 0; }
```

Comment:
Was there from the start, actually, just moved up one level in css files. Actually solves some issues with polymer right now. I'd rather leave it in, unless you can show me unexpected behavior. :)